articles/integrations/azure-api-management/configure-auth0.md

```diff
@@ -0,0 +1,84 @@
+---
+description: Configure Auth0 for use as an OAuth 2.0 server to authenticate users wanting access to an API managed by the Azure API Management service
+toc: true
+---
+
+# Configure Auth0
+
+To use Auth0 as an [OAuth 2.0 authorization server](/protocols/oauth2#oauth-roles), you'll need to execute the following setup steps:
+
+1. Create an Auth0 API and Non Interactive Client.
+2. Create a Connection to store your users.
+3. Create a user so that you can test your integration when you've finished setting it up.
+
+### Step 1: Create an API and Non Interactive Client
+
+An API is an entity that represents an external resource that's capable of accepting and responding to requests made by clients. You'll need to create an [Auth0 API](/apis) using the Management Dashboard to represent the API managed by Azure's API Management Service that you want secured by Auth0.
+
+You'll also need a [Non Interactive Client](/client), which represents your application and allows use of Auth0 for authentication. By default, when you create an API, Auth0 automatically creates an associated API. 
```

Comment:
This needs changing `By default, when you create an API, Auth0 automatically creates an associated API.` -> `By default, when you create an API, Auth0 automatically creates an associated non interactive client.`